live-editing/configs/GridConfigGenerator.ts

```diff
@@ -1264,6 +1265,17 @@ export class GridConfigGenerator implements IConfigGenerator {
             })
         }));
 
+        configs.push(new Config({
+            component: GridExportVisualizationComponent,
+            additionalFiles: ["/src/app/directives/prevent-scroll.directive.ts", "/src/app/data/utils.ts", "/src/app/data/invoiceData.ts"],
+            appModuleConfig: new AppModuleConfig({
+                imports: [GridExportVisualizationComponent, IgxGridModule, IgxPreventDocumentScrollModule],
+                ngDeclarations: [GridExportVisualizationComponent],
+                ngImports: [IgxPreventDocumentScrollModule, IgxGridModule],
+                ngProviders: []
+            })
+        }));
```

Comment:
Exporting services are missing and the export doesn't work in live editing. Same for tree grid config.

Reply:
Fixed